homeassistant/components/qnap/const.py

```diff
@@ -0,0 +1,7 @@
+"""The Qnap constants."""
+
+DEFAULT_NAME = "QNAP"
```

Comment:
Sorry, this variable is unused, let's keep it out of this PR until it's actually needed.
```suggestion
```